src/ui/components/ConfirmButton/index.js

```diff
@@ -118,8 +117,18 @@ export class ConfirmButtonBase extends React.Component<InternalProps, State> {
   }
 }
 
-const ConfirmButton: React.ComponentType<Props> = compose(translate())(
-  ConfirmButtonBase,
-);
+export const extractId = (ownProps: Props) => {
+  return `confirmButton-${ownProps.className}`;
```

Comment:
I wasn't totally clear about this - will this work?

Reply:
I guess i need to change this anyway since classname is not required :/

Reply:
mmh, do you really need an ID? I think you can write:

```
extractId: () => '',
```

in the HOC declaration.

Reply:
ahh! ok that will work i think. I wasn't clear if I should be passing in *something*..